README.md

```diff
@@ -28,7 +28,7 @@ Come on in, the water's lovely. More help? Ping `jfire` or `bhousel` on
 
 ## Prerequisites
 
-* [Node.js](http://nodejs.org/) version 4 or newer
+* [Node.js](http://nodejs.org/) version 4 or newer, ([npm](https://www.npmjs.com/package/npm) version 3 or newer)
```

Comment:
Oh, wait. You fixed that one in https://github.com/openstreetmap/iD/commit/348a973f688d2921f37d2cf5c4ef7cabbe7bc941. (Why you no tell me? :stuck_out_tongue_closed_eyes: )